File: convert_logo.py
import base64
import os

file_path = r'c:\Users\Usuario\Downloads\Recurso 8@1080x.png'
with open(file_path, 'rb') as image_file:
    encoded_string = base64.b64encode(image_file.read())
    print(encoded_string.decode('utf-8'))

File: convert_logo.js
const fs = require('fs');
const path = 'c:\\Users\\Usuario\\Downloads\\Recurso 8@1080x.png';
const buffer = fs.readFileSync(path);
const base64 = buffer.toString('base64');
fs.writeFileSync('logo_base64.txt', base64, 'utf8');
console.log('Done');
